assets/css/amp-stories.css

```diff
@@ -3,7 +3,3 @@ amp-story-grid-layer[template="fill"] .wp-block-image { margin: 0; }
 .has-background {
 	padding: 10px !important;
 }
-
-amp-story-grid-layer > * {
-	width: auto !important;
```

Comment:
@swissspidy I noticed that this CSS condition changes the frontend view and overrides some values needed with absolute positioning.

Would removing this break something that you worked on?

Reply:
Yeah I added it because of the background color options. The AMP runtime forces a 100% width for elements, making text blocks with a custom background color look odd.

I guess that won't be a problem anymore with this PR here.